walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -417,24 +418,24 @@ Verify that you followed each step in the procedure above.  If you are still hav
 
 === Demonstrationg offline support
 
-. On your mobile device:
+. Using the Firefox browser:
 .. Log into the showcase app.
-.. Activate airplane mode or disable network connectivity.
+.. Simulate offline mode. In Firefox, select *Work Offline* from the *Web Developers* menu.
 .. Create a new task.
```

Comment:
Tried to follow the steps in [1] - when I go offline and try to create new task, page for creating new task does not appear.

[1] https://routez5ynktzk-codeready.apps.dbizzarr-b9fe.open.redhat.com/tasks